src/main/scala/centaur/test/workflow/WorkflowData.scala

```diff
@@ -53,7 +53,7 @@ object WorkflowData {
   implicit class EnhancedPath(val path: Path) extends AnyVal {
     /** Read an entire file into a string, closing the underlying stream. */
     def slurp: String = {
-      val source = io.Source.fromFile(path.toFile, "UTF-8")
+      val source = scala.io.Source.fromFile(path.toFile, "UTF-8")
```

Comment:
Some people (i.e. me) enjoy having philosophical debates in PRs. Other people (i.e. most of the rest of the team) do not. People created ToL ("thinking out loud") as a code word for something which is not a blocker/todo on a PR but they'd like to air out for one reason or another.